session.go

```diff
@@ -19,6 +19,10 @@ import (
 // When Command() returns an empty slice, the user requested a shell. Otherwise
 // the user is performing an exec with those command arguments.
 //
+// By default, the Session translates NL to CRNL (`\n`=>`\r\n`). This
+// behavior can be disabled by adding a PtyCallback to the Server and
```

Comment:
PtyCallback takes a Pty struct (value type, not pointer), so the callback can't mutate anything its caller sees.

Thus this can't work.

Reply:
I suppose could you play context value games if you really wanted to use the PtyCallback, and add a context value on the callback's context that let you change that setting.

But that's pretty bizarre, even with an API helper "SetPtyCallbackDisableTranslation(context.Context)".